pytket/docs/index.rst

```diff
@@ -90,7 +88,7 @@ Licensed under the `Apache 2 License <http://www.apache.org/licenses/LICENSE-2.0
     
     Manual <https://cqcl.github.io/pytket/manual/index.html>
     Extensions <https://cqcl.github.io/pytket-extensions/api/index.html>
-    Example notebooks <https://github.com/CQCL/pytket/tree/main/examples>
+    Example notebooks <https://github.com/CQCL/pytket/tree/main/examples#pytket-examples>
```

Comment:
This link is still not working.

Reply:
I think its just the #pytket examples heading that changed

Reply:
works now at least